### PR TITLE
initial set-up

### PR DESCRIPTION
**Pull request by prodionov**
Reference #16

### CODE REVIEW

**Review by hasansaad1 — APPROVED** [review]

**Review by ghassanmas — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Mashrou Siha</title>
```
Comment (matthewdking) [inline index.html:5]: Isn't your project called rikaz?

**Review by matthewdking — COMMENTED** [review]
I think I would include some sort of dev server in this PR as well?

File: index.html (hunk 1 above)
Comment (prodionov) [reply index.html:5]: Rekaz is the name of the Department that coneys the survey. The project is called Mashrou Siha (Project health).

**Comment by hasansaad1** [comment]
@matthewdking added the webpack-dev-server package

**Review by matthewdking — APPROVED** [review]

**Pull request merged**